src/shared/components/communication-framework/ConversationIcon.tsx

```diff
@@ -31,24 +31,30 @@ type Props = {
 
 const ConversationIcon = (props: Props) => {
   const dispatch = useDispatch();
+  const analyticsRef = useRef<HTMLDivElement | null>(null);
```

Comment:
`analyticsRef` -> `elementRef` (it's a reference to an html element)

Reply:
That's done